### DISCUSSION TITLE
Typography Prose does not shrink

### DISCUSSION THREAD

**Discussion by Wulfheart** [tailwindlabs/tailwindcss#12279] [Help] [answered]
I'm currently working on a webpage, and I've encountered an issue with text shrinkage. Specifically, I'm having trouble getting the text inside a `<pre><code>` block of prose to shrink as expected. Strangely, this issue is isolated to the `<pre><code>` block, and everything else on the page seems to be functioning correctly.

I've included the relevant code snippet for reference:
https://play.tailwindcss.com/nV3RLBqdgR?layout=horizontal&size=1502x720

**Comment by wongjn** [comment, marked as answer]
The text would not shrink with default configuration, since it is set at a static `font-size`.

If you meant the containing elements not shrinking in width, it is because of the implicit `min-width: min-content` applied because of the horizontal flex layout. You could consider overriding this by applying `min-width: 0`: https://play.tailwindcss.com/rYZ34dwv8D?layout=horizontal&size=1502x720
